relays/bin-substrate/src/cli/bridge.rs

```diff
@@ -218,3 +221,137 @@ macro_rules! select_full_bridge {
+
+impl CliBridgeBase for RialtoToMillauCliBridge {
+	type Source = relay_rialto_client::Rialto;
+	type Target = relay_millau_client::Millau;
+}
+
+impl CliBridge for RialtoToMillauCliBridge {
+	type Finality = crate::chains::rialto_headers_to_millau::RialtoFinalityToMillau;
+}
+
+//// `Westend` to `Millau` bridge definition.
+pub struct WestendToMillauCliBridge {}
+
+impl CliBridgeBase for WestendToMillauCliBridge {
+	type Source = relay_westend_client::Westend;
+	type Target = relay_millau_client::Millau;
+}
+
+impl CliBridge for WestendToMillauCliBridge {
+	type Finality = crate::chains::westend_headers_to_millau::WestendFinalityToMillau;
+}
+
+//// `Rococo` to `Wococo` bridge definition.
+pub struct RococoToWococoCliBridge {}
+
+impl CliBridgeBase for RococoToWococoCliBridge {
+	type Source = relay_rococo_client::Rococo;
+	type Target = relay_wococo_client::Wococo;
+}
+
+impl CliBridge for RococoToWococoCliBridge {
+	type Finality = crate::chains::rococo_headers_to_wococo::RococoFinalityToWococo;
+}
+
+//// `Wococo` to `Rococo` bridge definition.
+pub struct WococoToRococoCliBridge {}
+
+impl CliBridgeBase for WococoToRococoCliBridge {
+	type Source = relay_wococo_client::Wococo;
+	type Target = relay_rococo_client::Rococo;
+}
+
+impl CliBridge for WococoToRococoCliBridge {
+	type Finality = crate::chains::wococo_headers_to_rococo::WococoFinalityToRococo;
+}
+
+//// `Kusama` to `Polkadot` bridge definition.
+pub struct KusamaToPolkadotCliBridge {}
+
+impl CliBridgeBase for KusamaToPolkadotCliBridge {
+	type Source = relay_kusama_client::Kusama;
+	type Target = relay_polkadot_client::Polkadot;
+}
+
+impl CliBridge for KusamaToPolkadotCliBridge {
+	type Finality = crate::chains::kusama_headers_to_polkadot::KusamaFinalityToPolkadot;
+}
+
+//// `Polkadot` to `Kusama`  bridge definition.
+pub struct PolkadotToKusamaCliBridge {}
+
+impl CliBridgeBase for PolkadotToKusamaCliBridge {
+	type Source = relay_polkadot_client::Polkadot;
+	type Target = relay_kusama_client::Kusama;
+}
+
+impl CliBridge for PolkadotToKusamaCliBridge {
+	type Finality = crate::chains::polkadot_headers_to_kusama::PolkadotFinalityToKusama;
+}
+
+//// `Millau` to `RialtoParachain`  bridge definition.
+pub struct MillauToRialtoParachainCliBridge {}
+
+impl CliBridgeBase for MillauToRialtoParachainCliBridge {
+	type Source = relay_millau_client::Millau;
+	type Target = relay_rialto_parachain_client::RialtoParachain;
+}
+
+impl CliBridge for MillauToRialtoParachainCliBridge {
+	type Finality =
+		crate::chains::millau_headers_to_rialto_parachain::MillauFinalityToRialtoParachain;
+}
+
+//// `RialtoParachain` to `Millau` bridge definition.
+pub struct RialtoParachainToMillauCliBridge {}
+
+impl CliBridgeBase for RialtoParachainToMillauCliBridge {
+	type Source = relay_rialto_parachain_client::RialtoParachain;
+	type Target = relay_millau_client::Millau;
+}
+
+//// `WestendParachain` to `Millau` bridge definition.
+pub struct WestendParachainToMillauCliBridge {}
```

Comment:
Let's rename it to `WestmintToMillauCliBridge` maybe? There are many parachains on Westend and iiuc this struct is dedicated to Westmint.

Reply:
Done.